.github/actions/draco-install-dep/action.yml

```diff
@@ -50,7 +41,8 @@ runs:
         -DBUILD_SHARED_LIBS=OFF
         -DDRACO_JS_GLUE=OFF
         -DCMAKE_BUILD_TYPE=Release
-        -DCMAKE_INSTALL_PREFIX:PATH=../draco_install
+        -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+        -DCMAKE_INSTALL_PREFIX=../draco_install
```

Comment:
how does that fix the installation issue ?

Reply:
No idea :)  
I read the CMake doc but I don't understand.  
Looks like adding `:PATH` causing issues.